ui/guest/src/components/Guest.tsx

```diff
@@ -1113,11 +1131,11 @@ export function Guest(props) {
     return ElementRegistry.register(payload);
   }
 
-  function deregister(id) {
+  function deregister(id: string) {
     return ElementRegistry.deregister(id);
   }
 
-  function onEvent(event, dispatcher) {
+  function onEvent(event: Event, dispatcher) {
```

Comment:
- Type dispatcher
- Return type

Reply:
@rart there is no return on the if and there's a return true on the else, what should the return type be?